docs/site/Validation.md

```diff
@@ -0,0 +1,28 @@
+---
+lang: en
+title: 'Validation'
+keywords: LoopBack 4.0, LoopBack 4
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Validation.html
+---
+
+Within a LoopBack application, validation can be added in various places
+depending on the usage. Some types of validations come out-of-the-box in
+LoopBack, such as type validation in the REST layer, whereas some require
+additional configuration or code.
+
+For illustration purpose, a `CoffeeShop` model is being used. It has the
```

Comment:
Do we want to link the package here? Cause We seem to use several examples from CoffeeShop

Reply:
i'm using my own https://github.com/dhmlau/loopback4-validation-example. :( 
Not sure if it's a good idea to link my personal repo here. Any suggestions?

Reply:
Perhaps for now. But eventually, it would be cool to see it as an example app?

Reply:
adding the validation example in https://github.com/strongloop/loopback-next/pull/4895